### PR TITLE
improve(LM rewards): uUSDwETH-DEC week 17 liquidity mining rewards

### PR DESCRIPTION
**Pull request by chrismaree**
This PR adds week 17's liquidity mining payouts for uUSDwETH-DEC. The script was run from 2020-11-16 23:00:00 UTC (11271761) to 2020-11-23 23:00:00 UTC (11317321). 

**Note** that the payouts for this week's rewards are the second LM rewards who's output is based on the new [UMA Dev mining program](https://medium.com/uma-project/uma-announces-developer-mining-6f6fe15d5604). Based off the dev mining program, half of all rewards allocated to the uUSDwETH contract, pro-rata to other contracts within the UMA dev mining program, are allocated as liquidity mining rewards. 

To see the output of this week's dev mining see PR https://github.com/UMAprotocol/protocol/pull/2216.

From this week's dev mining, a total of `29381.306609825289708292` UMA is allocated to the uUSDwETH contract. As UMA has decided to send half of these rewards as LM rewards, the total allocated for these weekly payouts is **14690 UMA** for uUSDwETH.

**To re-generate these results run:**
```
node ./liquidity-mining/CalculateBalancerLPRewards.js --fromBlock 11271761 --toBlock 11317321 --poolAddress="0xcce41676a4624f4a1e33a787a59d6bf96e5067bc" --umaPerWeek=14690 --tokenName="yusdeth" --week=17 --network mainnet_mnemonic
```

### CODE REVIEW

File: packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/Week_17_Mining_Rewards.json [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    "0x181c2d506b4aad984e3a98f09ac8eaf895671599": "131.168408056614929086",
+    "0x186cf5714316f47bc59e30a850615a3f938d7d79": "144.324305676187406122",
+    "0x18db8b99c1d6e439fa44fd87bfb1109e345e98da": "14.938398734578934199",
+    "0x1dd50763ebf30966922c428177b090a90a634de9": "9.156741383956258899",
+    "0x1e4bda16734c1f9e86a66e310fcf8bd3c4f9c0f0": "359.648163704539283323",
+    "0x1ee10aeaebc6336b813daf9067248fa90de2b589": "46.041856405414321816",
+    "0x1f295ff3ea09d8eac8a39e3617bd4646d4df7c9a": "120.171047337870375806",
+    "0x230e7f448f6ebc7c4433cce03ee92eeb81681a2c": "198.804194674639862899",
+    "0x25125e438b7ae0f9ae8511d83abb0f4574217c7a": "212.04219245372769194",
+    "0x25ae7b45d8646580dfcae403d29164729eb8642f": "2452.530513816210079689",
```
Comment (nicholaspai) [inline packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/Week_17_Mining_Rewards.json:24]: Another big one here just want to make sure you sanity checked

**Review by nicholaspai — APPROVED** [review]

File: packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/Week_17_Mining_Rewards.json (hunk 1 above)
Comment (chrismaree) [reply packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/Week_17_Mining_Rewards.json:24]: yup, I did. this is the biggest sponsor:
image

**Comment by hlambur** [comment]
Rewards for uUSDwETH-DEC send with txs: https://etherscan.io/tx/0x0b03eafaaa5170e76a025a20f5c5a120a2321c4c331e855d19b8254fed6f4253
https://etherscan.io/tx/0x945fc1bbc6e1afed00c02b7e1acaa23bed3a63f03b8ff6966b6a208144f9dba6

**Comment by hlambur** [comment]
Rewards for uUSDrBTC-DEC sent with tx: https://etherscan.io/tx/0x5137a9401e588180d1c928415ee6ec8c24087542768fc758e8178345883fee7a

**Pull request merged**